bundles/org.openhab.io.hueemulation/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -51,7 +51,7 @@
 			<description>All items that are tagged with the given tags are ignore by the Hue Emulation Service. Use commas to separate multiple entries.</description>
 			<default>internal</default>
 		</parameter>
-		<parameter name="discoveryIp" type="text" required="false">
+		<parameter name="discoveryIps" type="text" required="false">
```

Comment:
I think the fix should be the other way around. The config field name should be updated instead. This config is only supposed to handle a single IP address right?

Reply:
Also, changing this would not be backwards compatible for people already using the binding

Reply:
No, the code is able to handle multiple IPs (ConfigStore.java): 

```
        if (config.discoveryIps != null) {
            discoveryIps = Collections.unmodifiableSet(Stream.of(config.discoveryIps.split(",")).map(String::trim)
                    .map(this::byName).filter(e -> e != null).collect(Collectors.toSet()));
```

it is also documented as such:

README.md
> This option allows you to override what addresses are used for the announcement.
> You can have multiple comma separated entries.
> 
> ```
> org.openhab.hueemulation:discoveryIps=192.168.1.100,::FFFF:A9DB:0D85
> ```

There is also code in UpnpServer dealing with multiple IPs.

I personally feel not confident reverting it back to a single ip usage as there is some code evolved in the background where I dont really understand what it does. I try though to figure out how to make a fallback read so the old value is not lost.

Reply:
If it was designed to support multiple IPs then you can keep that feature. It just wasn't clear from either the param name or the description that a comma-separated list was supported. I only suggested changing the config field because I though it only supported a single IP address.

Reply:
Indeed the text inside the field does not make this clear. I also just saw it when I looked into the code (and the services readme). I am not sure if the text facing the user should be changed as well. What do you say?

Reply:
Yes, please update the description to make this clear.